source/agora/consensus/data/genesis/Coinnet.d

```diff
@@ -33,6 +33,7 @@ public immutable Block GenesisBlock =
         enrollments: Enrollments,
         validators:  BitField!ubyte(6),
         signature:   Signature.init,
+        time_offset: 0, // This is offset in seconds from Genesis time
```

Comment:
We could just say "By convention" or similar here, since it seems weird to set the offset from genesis in the genesis block ?

Reply:
Updated